Automattic/jetpack
# Make "My Jetpack" the first sub-menu item, above "Dashboard".

**[WIP] - Work in progress**

This diff makes **My Jetpack** the default page that shows when **Jetpack** is clicked in the left sidebar in WP Admin.
( p1HpG7-mMC-p2#comment-63646 )(See before & after screenshots below) 

@eeeeeevon2023 & @p3ob7o was hoping to get this shipped in the July release, but it didn't happen. So we're hoping to get this in before the **Aug 16th code freeze**

**BEFORE** | **AFTER**
--- | ---
![Markup 2023-07-22 at 08 06 37](https://github.com/Automattic/jetpack/assets/11078128/95d9614c-1a0c-40d0-a3a4-12f8252efe3a) | ![Markup 2023-07-22 at 08 14 05](https://github.com/Automattic/jetpack/assets/11078128/8f400137-ad11-4e63-b426-ee46e226e972)

Asana task: 1161179020265237-as-1205110464380103/f





## Proposed changes:


* Makes the "**Jetpack**" menu item link to `?page=my-jetpack` instead of `?page=jetpack#/dashboard`
* Moves the "**My Jetpack**" sub-menu item to first position, making the "**Dashboard**" sub-menu item be in the second position.
* Updates the jQuery that controls which sub-menu item is currently active/highlighted to account for the re-positioned **My-Jetpack** sub-menu link.

### Other information:

- [] Have you written new tests for your changes, if applicable?

## Jetpack product discussion



p1HpG7-mMC-p2#comment-63646

## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






- Do one of these:
    - Checkout this PR and spin up a local development build of Jetpack, OR
    - Spin up a Jurassic.ninja or Ephemeral site with the Jetpack Beta plugin running this PR.
- Connect Jetpack.
- In wp-admin, click the "**Jetpack**" side menu item.
- Verify you are taken to the "My Jetpack" page.
- View the Jetpack side menu. Verify "My Jetpack" is the first menu item, with "Dashboard" below it.
- Verify the "My Jetpack" sub-menu item takes you to the "My Jetpack" page, and the "Dashboard" sub-menu item takes you to the "Dashboard" page.
- Verify that when on the "My Jetpack" page the "My Jetpack" sub-menu item is active/highlighted, and when on the "Dashboard" page the "Dashboard" sub-menu item is highlighted.
